src/main/java/com/minecolonies/coremod/commands/colonycommands/ListColoniesCommand.java

```diff
@@ -59,24 +60,34 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
     {
         int page = 1;
-        final List<Colony> colonies = ColonyManager.getColonies();
+        int abandonedSince = 0;
+        if (args.length != 0)
+        {
+            page = getIthArgument(args, 0, 1);
+        }
+
+        if(args.length > 1)
+        {
+            abandonedSince = getIthArgument(args, 1, 0);
+        }
+
+        final List<Colony> colonies;
+        if(abandonedSince > 0)
+        {
+           colonies = ColonyManager.getColonies(abandonedSince);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)